general/Jupyterhub/jupyter/full_gpu_sbatch_jupyter.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash  --login
```

Comment:
Thanks for the updates, let's make this a shared GPU node with 1 GPU instead of a full node, to encourage that usage. I don't think we need to add another script for a full GPU node since that will be an easy change for the user.